src/global-firestore-data/courses.ts

```diff
@@ -0,0 +1,22 @@
+import { doc, getDoc } from 'firebase/firestore';
+import { coursesCollection, availableRostersForCoursesCollection } from '../firebase-config';
+
+export const getCourse = async (roster: string, subject: string, number: string) => {
```

Comment:
Instead of accepting a roster as a string like `FA20`, can this accept the season (of type `FirestoreSemesterSeason`), and year?

Reply:
> 

Yeah, I can do that